db/seeds.rb

```diff
@@ -1,7 +1,54 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'json'
+File.open("db/schools_data.json").each do |line|
+  parsed_course = JSON.parse(line)
+  # Every line in a the .txt file represents an JSON course object with
+  # belonging time objects
+  department = parsed_course["department"]
```

Comment:
Indent right side for nicer formatting

Reply:
Do you mean by putting the second part of the comment(line 5) on line 4? Then it breaks 80 characters. If it's about line 6, please explain how.

Reply:
Oh, I understand now